lib/msal-common/src/cache/entities/CredentialEntity.ts

```diff
@@ -82,11 +82,11 @@ export class CredentialEntity {
     generateType(): number {
         switch (this.credentialType) {
```

Comment:
Since the known-cases in these switch statements have no behavior logic, they are essentially mapping Credential Types/Authority types to cache value types. Would using a map instead of a switch statement be a better approach? You could still handle the default case by checking if the result of accessing a key on the object/map is undefined.